recovery/Dockerfile

```diff
@@ -1,9 +1,15 @@
-FROM node:16
+FROM node:19
```

Comment:
It might be worth using `node:18` if that does what you need. Version 18 is an LTS release, with support into 2025. Version 19 isn't LTS, and goes EOL this coming June (2023).

Reply:
Confirmed: https://www.geeksforgeeks.org/what-are-lts-versions-of-node-js/
I concur with @forevermatt 's suggestion. :)

Reply:
So, strange that a later version gets EOL before an older version.

Reply:
Yeah, one of the peculiarities about NodeJS versions is that even major versions are LTS and odd major versions aren't.